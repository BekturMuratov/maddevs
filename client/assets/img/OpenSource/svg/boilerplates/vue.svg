<svg xmlns="http://www.w3.org/2000/svg" width="336" height="125" fill="none" xmlns:v="https://vecta.io/nano"><g clip-path="url(#A)"><path d="M206.784.001l-38.797 67.203L129.191.001H-.008l167.996 290.997L335.984.001h-129.2z" fill="#41b883"/><path d="M206.785.001l-38.797 67.203L129.191.001h-62l100.798 174.595L268.786.001h-62.001z" fill="#34495e"/></g><defs><clipPath id="A"><path fill="#fff" d="M0 0h336v291H0z"/></clipPath></defs></svg>