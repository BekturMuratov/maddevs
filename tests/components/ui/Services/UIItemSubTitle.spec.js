import { render, screen } from '@testing-library/vue'
import UIItemSubTitle from '@/components/ui/Services/UIItemSubTitle'

const props = {
  itemSubTitle: 'some title',
}

describe('uIButton component', () => {
  it('should render correctly with slot', () => {
    const { container } = render(UIItemSubTitle, {
      props,
    })

    expect(screen.getByText(props.itemSubTitle)).not.toBeNull()
    expect(container).toMatchSnapshot()
  })
})
