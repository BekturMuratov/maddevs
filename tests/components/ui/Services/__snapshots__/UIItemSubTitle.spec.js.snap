// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UIButton component should render correctly with slot 1`] = `
<div>
  <h4
    class="item-sub-title"
  >
    some title
  </h4>
</div>
`;
